modules/@apostrophecms/doc-type/ui/apos/components/AposDocMoreMenu.vue

```diff
@@ -114,6 +120,9 @@ export default {
     },
     canDismissSubmission() {
       this.menu = this.recomputeMenu();
+    },
+    disableSaveDraft() {
```

Comment:
Why do we need a new prop and watcher if we're already watching `isPublished` and `isModified`?